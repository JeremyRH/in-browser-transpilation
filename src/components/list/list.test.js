import React from "react";
import renderer from "react-test-renderer";
import List from "./list.js";

describe("List component", () => {
    test("should be defined", () => {
        expect(List).toBeDefined();
    });
    test("snapshot", () => {
        expect(renderer.create(<List initialItems={["a", "b", "c"]} />).toJSON()).toMatchSnapshot();
    });
});
