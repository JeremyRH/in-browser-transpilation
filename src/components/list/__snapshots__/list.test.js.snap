// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`List component snapshot 1`] = `
<div
  className="list"
>
  <ul>
    <li>
      a
    </li>
    <li>
      b
    </li>
    <li>
      c
    </li>
  </ul>
  <input
    onInput={[Function]}
  />
  <button
    onClick={[Function]}
  >
    Add
  </button>
</div>
`;
